packages/rollup-plugin/src/index.ts

```diff
@@ -8,24 +8,28 @@ import {
   virtualCssFileFilter,
 } from '@vanilla-extract/integration';
 import { relative, normalize, dirname } from 'path';
+import { createFilter, FilterPattern } from '@rollup/pluginutils';
 
 interface Options {
   identifiers?: IdentifierOption;
   cwd?: string;
+  exclude?: FilterPattern;
+  include?: FilterPattern;
 }
 export function vanillaExtractPlugin({
   identifiers,
   cwd = process.cwd(),
+  exclude,
+  include = cssFileFilter,
 }: Options = {}): Plugin {
   const emittedFiles = new Map<string, string>();
   const isProduction = process.env.NODE_ENV === 'production';
+  const filter = createFilter(include, exclude);
 
   return {
     name: 'vanilla-extract',
     async transform(_code, id) {
-      if (!cssFileFilter.test(id)) {
-        return null;
-      }
+      if (!filter(id)) return null;
```

Comment:
Just defaulting the `include` to `cssFileFilter` seems like a foot gun. It seems reasonable that a user would want to scope their rules to a folder for example. e.g. `include: ['vanilla-styles/**/*']`. But this would lead to plugin running on all files in that folder, not just the `.css.ts` files. I think the filter function should be combined with the regular `cssFileFilter` check as all files still need to have a `.css.ts/js` extension, that's not configurable.